File: tests/cgi/cgi-bin/cgi_get.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    cgi_get.py                                         :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: maricard <maricard@student.porto.com>      +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2023/10/18 17:24:21 by maricard          #+#    #+#              #
#    Updated: 2023/10/21 16:12:20 by maricard         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

from datetime import datetime as date

# get current date and time, time not parsed
date_and_time = date.now()

# parse to format dd/mm/YY H:M:S
parsed_value = date_and_time.strftime("%d/%m/%Y %H:%M:%S")

# print parsed value
print(parsed_value)

File: tests/cgi/cgi-bin/cgi_get.js
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   cgi_get.js                                         :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: maricard <maricard@student.porto.com>      +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2023/10/18 17:24:21 by maricard          #+#    #+#             */
/*   Updated: 2023/11/12 12:18:17 by maricard         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

// Get current date and time
const dt = new Date();

// Format to dd/mm/YYYY H:M:S
const DateOptions = {
  day: '2-digit',
  month: '2-digit',
  year: 'numeric'
};

const TimeOptions = {
	hour: '2-digit',
	minute: '2-digit',
	second: '2-digit',
	hour12: false
  };

const formattedDT = dt.toLocaleDateString('en-GB', DateOptions) + ' ' + 
					dt.toLocaleTimeString('en-GB', TimeOptions);

const message = formattedDT;

console.log(message);
